packages/gatsby-theme-learning/src/hooks/use-attempt.ts

```diff
@@ -71,6 +77,10 @@ export const useAttempt = (fetchAttemptParams: FetchAttemptParams) => {
           }
           throw new Error();
         } else {
+          // we invalidate course status cache as the user might be just being
+          // enrolled into the course so we want fresh course status data to be fetched
+          // see https://github.com/commercetools/commercetools-docs-kit/issues/1644
+          invalidateCache();
```

Comment:
@FFawzy  okay for now but my learning is that the API proxy (server) side "knows" much better when other resources might be stale or not. In this case, the API tier "knows" that it auto-enrolled and had to create an attempt for the first time, so it knows that the course is stale - but the client can only guess. 

So if this kind of situation becomes more commonplace we could consider rather letting the API hint to the client what other resources to mark stale now vs the client having to guess.